assets/css/custom.scss

```diff
@@ -0,0 +1,27 @@
+---
+---
+
+$standard-spacing: 20px;
+
+footer {
+    font-size: 0.75em;
+    padding: $standard-spacing;
+    margin-top: $standard-spacing;
+    background-color: lightgray;
+
+    h2 {
+        font-size: 1.5em;
+        text-align: center;
+    }
+}
+
+header.texture-custom {
+    background-color: cornflowerblue;
+    background-image: none;
+    margin-bottom: $standard-spacing;
+}
+
+a {
+    color: cornflowerblue;
+    text-decoration: underline;
+}
```

Comment:
Since there isn't a lot of sass in this repo (since the bulk of it is from the `texture` template), I think we can just keep all template-override scss in this `custom.scss`

Reply:
Yeah, certainly if it's easier and more convenient.